tools/clang/unittests/SPIRV/WholeFileCheck.cpp

```diff
@@ -202,28 +198,24 @@ WholeFileTest::getAbsPathOfInputDataFile(const std::string &filename) {
   return path;
 }
 
-bool WholeFileTest::runWholeFileTest(std::string filename, bool generateHeader,
+void WholeFileTest::runWholeFileTest(std::string filename, bool generateHeader,
                                      bool runSpirvValidation) {
   inputFilePath = getAbsPathOfInputDataFile(filename);
 
-  bool success = true;
-
   // Parse the input file.
-  success = success && parseInputFile();
+  ASSERT_TRUE(parseInputFile());
```

Comment:
`ASSERT_TRUE` guarantees that if this expression returns false, the following code will not be executed. It makes sense to use `ASSERT_*` here b/c if input file is wrong, there is no need to proceed.